deploy/templates/nstemplatetiers/appstudio/ns_tenant.yaml

```
@@ -262,6 +262,13 @@ objects:
   - kind: ServiceAccount
     namespace: ${SPACE_NAME}-tenant
     name: appstudio-pipeline
+- apiVersion: v1
+  data: {}
+  kind: ConfigMap
+  metadata:
+    labels:
+      config.openshift.io/inject-trusted-cabundle: "true"
+    name: trusted-ca
```

should we add the namespace ? 
```suggestion
    name: trusted-ca
    namespace: ${SPACE_NAME}-tenant
```

---

added. thanks!